src/Protocol/DFRN.php

```diff
@@ -2603,8 +2603,8 @@ private static function processEntry($header, $xpath, $entry, $importer, $xml)
 		$item["parent-uri"] = $item["uri"];
 
 		$inreplyto = $xpath->query("thr:in-reply-to", $entry);
-		if (!empty($inreplyto->item[0])) {
-			foreach ($inreplyto->item[0]->attributes as $attributes) {
+		if (is_object($inreplyto->item(0))) {
+			foreach ($inreplyto->item(0)->attributes as $attributes) {
```

Comment:
Wait, what is this notation?

Reply:
```item``` is no array, it is an object that you address via the integer value. It is done at many places all over the system - and is working fine.

Reply:
I still don't understand, is it a method invocation? A property retrieval?

Reply:
I must confess, I don't know. But I know that it works ;-)

Reply:
*sigh*

Reply:
After a cursory research, it actually is a method: http://php.net/manual/en/domnodelist.item.php

Reply:
It's a method invocation. But its return value implement interface traversable to be accessed as array. https://secure.php.net/manual/en/class.traversable.php

Reply:
@MrPetovan I just wanted to post that link as well.

Reply:
🙌

(my girlfriend is getting suspicious about us two)